tests/ciscotest.rb

```diff
@@ -172,6 +172,34 @@ def validate_property_excluded?(feature, property)
     !node.cmd_ref.supports?(feature, property)
   end
 
+  def skip_incompat_version?(feature, property)
+    ov = node.os_version_get(feature, property)
+    return if ov.nil?
+    im = Platform.image_version
+    pid = node.product_id
+
+    array = ov.split(',')
+    hash = {}
+    array.each do |e|
+      key_value = e.split(':')
+      hash[key_value[0].strip] = key_value[1]
+    end
+
+    match = nil
+    hash.keys.each do |prid|
```

Comment:
It would be good to use our `product_tag` api (https://github.com/cisco/cisco-network-node-utils/blob/develop/tests/ciscotest.rb#L466) as a way to make sure any product tags in the yaml file are supported.

This way we provide input validation if developers try to use a tag this not supported.